frontend/src/views/Collection/index.tsx

```diff
@@ -185,79 +200,134 @@ const Collection: FC = () => {
       <Head>
         <title>cellxgene | {collection.name}</title>
       </Head>
-      <ViewGrid isFilterEnabled={isFilterEnabled}>
-        {collection.has_revision && visibility === VISIBILITY_TYPE.PRIVATE && (
-          <StyledCallout intent={Intent.PRIMARY} icon={null}>
-            <span data-test-id="revision-status">
-              {collection.revision_diff
-                ? "This collection has changed since you last published it."
-                : "This is a private revision of a public collection."}
-            </span>
-          </StyledCallout>
-        )}
-
-        {!isRevision && <Banner collectionId={collection.id} />}
-
-        <CollectionInfo>
-          <H3 data-test-id="collection-name">{collection.name}</H3>
-          <Description data-test-id="collection-description">
-            {collection.description}
-          </Description>
-          <LinkContainer>
-            {/*
-             * (thuang): Contact and Links order defined here:
-             * https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/124
-             */}
-            {renderContact(collection.contact_name, collection.contact_email)}
-            {renderLinks(collection.links)}
-          </LinkContainer>
-        </CollectionInfo>
-
-        {shouldShowPrivateWriteAction && (
-          <ActionButtons
-            id={id}
-            addNewFile={addNewFile}
-            isPublishable={isPublishable}
+      {isFilterEnabled ? (
```

Comment:
Hey @tihuan I agree this impacts readability, but I am keen to avoid touching the current code to reduce the risk of introducing defects. My preference would be to leave the ternary as is, and avoid moving the current functionality into a new component (with prop drilling) particularly as the ternary will no longer be needed once we remove the filter feature flag. Would that be ok with you? 🐡

Reply:
Yeah that's a great point! Since the OldView is temp code at this point, probably no big deal 😆 Let's just keep it as is 🙆‍♂️ Thank you!

Reply:
Thank you so much @tihuan!